app/src/ttrpc.rs

```diff
@@ -7,10 +7,9 @@ use super::*;
 use ::ttrpc::Server;
 use std::path::Path;
 
-const DEFAULT_KEYPROVIDER_UNIX_ADDR: &str =
-    "unix:///opt/confidential-containers/attestation-agent/keyprovider.sock";
-const DEFAULT_GETRESOURCE_UNIX_ADDR: &str =
-    "unix:///opt/confidential-containers/attestation-agent/getresource.sock";
+const DEFAULT_UNIX_SOCKET_DIR: &str = "/run/confidential-containers/attestation-agent";
+const DEFAULT_KEYPROVIDER_SOCKET_ADDR: &str = "unix:///run/confidential-containers/attestation-agent/keyprovider.sock";
```

Comment:
Maybe we could use https://crates.io/crates/const_format and avoid duplicating strings?